console/v1/types_console_plugin.go

```diff
@@ -50,6 +50,9 @@ const (
 	// Lazy wont preload any plugin's localization resources, instead
 	// will leave thier loading to runtime's lazy-loading.
 	Lazy LoadType = "Lazy"
+	// Empty is the default value of the LoadType field and it's
+	// purpose is to improve discoverability of the field.
+	Empty LoadType = ""
```

Comment:
What is the behaviour when this is empty?

Just noticed that the docs on LoadType are a little light, can we expand those?

```
// loadType indicates how the plugin's localization resource should be loaded.
// Valid values are Preload, Lazy and the empty string.
// When set to Preload, all localization resources are fetched when the plugin is loaded.
// When set to Lazy, localization resources are lazily loaded as and when they are required by ...
// When omitted or set to the empty string, the behaviour is equivalent to ...
```